Dockerfiles/WebApp

```diff
@@ -10,28 +10,36 @@ ARG APP=target/webapp
 ENV JAVA_HOME /opt/java/openjdk
 COPY --from=jdk $JAVA_HOME $JAVA_HOME
 
-# Will be replaced on Jetty in 5.26
-# Copy Tomcat
-ENV CATALINA_HOME /usr/local/tomcat
-COPY --from=tomcat:9-jre11-temurin-focal $CATALINA_HOME $CATALINA_HOME
+ENV OPENL_APP /openl-app
+ENV JETTY_HOME $OPENL_APP
+ENV JETTY_BASE $OPENL_APP
+ENV OPENL_LIB /openl-lib
```

Comment:
Think about using the following dir structure:

/opt/openl/bin (application)
/opt/openl/lib (libraries)